eodatasets/prepare/s2_l1c_aws_pds_prepare.py

```diff
@@ -96,6 +96,7 @@ def valid_region(images, mask_value=None):
 
 
 def _to_lists(x):
+    # type (Tuple) -> List
```

Comment:
It needs a colon after "type" to be parsed as a type hint:
```suggestion
    # type: (Tuple) -> List
```